snapshot/lib/snapshot/detect_values.rb

```diff
@@ -19,6 +19,10 @@ def self.set_additional_default_values
         end
       end
 
+      if !config[:test_without_building].nil? && config[:derived_data_path].nil?
```

Comment:
Use `config[:derived_data_path].to_s.length == 0` instead, as an empty string should also show this error message